### PR TITLE
Generating latest spec

### PR DESCRIPTION
**Pull request by mehm23**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by hovsepm** [comment]
@mehm23 please walk over Breaking Changes and Model Validator errors and fix them. Since you are making changes in the stable swagger there should be no breaking changes.

**Comment by praries880** [comment]
@mehm23 have these changes been trough an A/AzureAPI review?

File: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json [hunk 1]
```diff
@@ -4061,7 +3991,7 @@
       }
     },
     "BEKDetails": {
-      "description": "BEK is bitlocker encryption key.",
+      "description": "BEK is bitlocker encrpytion key.",
```
Comment (KrisBash) [inline specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json]: Spelling

File: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json [hunk 2]
```diff
@@ -3909,7 +3839,7 @@
       "type": "object",
       "properties": {
         "objectType": {
-          "description": "This property will be used as the discriminator for deciding the specific types in the polymorphic chain of types.",
+          "description": "This property will be used as the discriminator for deciding the specific types in the polymorhpic chain of types.",
```
Comment (KrisBash) [inline specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json]: Spelling

File: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json [hunk 3]
```diff
@@ -3677,7 +3611,7 @@
       "x-ms-discriminator-value": "AzureWorkloadSQLPointInTimeRestoreRequest"
     },
     "AzureWorkloadSQLRecoveryPoint": {
-      "description": "SQL specific recovery point, specifically encapsulates full/diff recovery point along with extended info",
+      "description": "SQL specific recoverypoint, specifcally encaspulates full/diff recoverypoint alongwith extended info",
```
Comment (KrisBash) [inline specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json]: Spelling and spacing errors

File: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json [hunk 4]
```diff
@@ -3464,8 +3419,44 @@
...
+          "format": "date-time",
+          "description": "PointInTime value",
+          "type": "string"
+        }
+      },
+      "x-ms-discriminator-value": "AzureWorkloadPointInTimeRestoreRequest"
+    },
     "AzureWorkloadRecoveryPoint": {
-      "description": "Workload specific recovery point, specifically encapsulates full/diff recovery point",
+      "description": "Workload specific recoverypoint, specifcally encaspulates full/diff recoverypoint",
```
Comment (KrisBash) [inline specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json]: spelling "specifically"

File: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json [hunk 5]
```diff
@@ -6090,7 +6025,7 @@
       "type": "object",
       "properties": {
         "objectType": {
-          "description": "This property will be used as the discriminator for deciding the specific types in the polymorphic chain of types.",
+          "description": "This property will be used as the discriminator for deciding the specific types in the polymorhpic chain of types.",
```
Comment (KrisBash) [inline specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json]: Spelling - "polymorphic" for all cases

File: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-08-10/operations.json [hunk 6]
```diff
@@ -120,7 +119,7 @@
       "type": "object",
       "properties": {
         "value": {
-          "description": "List of available operations.",
+          "description": "List of available operationss.",
```
Comment (KrisBash) [inline specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-08-10/operations.json]: Please check spelling throughout. "operations"

**Review by KrisBash — CHANGES_REQUESTED** [review]
Changes to add the "put" api and SAP definitions appear to be ok for ARM. Numerous spelling errors were introduced in these commits and should be corrected for the benefit of documentation quality.

**Comment by praries880** [comment]
@KrisBash Do the latest changes take care of your review comments?

**Review by KrisBash — APPROVED** [review]
LGTM

**Pull request merged**